Aura Operating System/Aura OS/System/Utils/Bug.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Aura_OS.System.Utils
+{
+    class Bug
+    {
+        public int IssuesNumber = 0;
+
+        public void ScanIssues()
+        {
+            //How to create a new type of error ?
+            //0 x ? ?
+            //0 -> I don't know why, just for the style
+            //x -> Same.
+            //?? -> First: Level of error 0-9; Second: Number of the problem;
+
+            if (!Directory.Exists(@"0:\System"))
+            {
+                IssuesNumber++;
+                Crash.StopKernel("Error 0x91: Aura is not installed anymore");
```

Comment:
I guess it will be better to don't use Crash.StopKernel but "throw new exception".

Reply:
All exceptions stop the kernel ?

Reply:
Yep

Reply:
Oh okay, I will change that in the next commit :-)